dask/array/core.py

```diff
@@ -2660,6 +2660,10 @@ def bincount(x, weights=None, minlength=None):
 
     return Array(dsk, name, chunks, dtype)
 
+@wraps(np.digitize)
+def digitize(a, bins, right=False):
+    func = partial(np.digitize, bins=bins, right=right)
+    return elemwise(func, a)
```

Comment:
I see. It appears to me map_partitions is more descriptive than map_blocks. 

It is under my impressions that astronomers cannot live very well with Pandas DataFrames because DataFrames do not support vector columns. Please don't abandon the array interface.